spec/features/__snapshots__/test_results_successful_specs-rspec-3.8.gemfile.snap

```diff
@@ -0,0 +1,25 @@
+[
+  [0] {
+    "test_result" => {
+            "status" => {
+        "type" => "success"
+      },
+                "id" => "./spec/fixture_specs/successful_specs.rb[1:1]",
+      "display_name" => "a successful group has a successful test",
+            "output" => nil,
+           "runtime" => "<cleaned for test>",
+              "tags" => [],
+              "meta" => {},
+          "location" => {
+        "file" => "./spec/fixture_specs/successful_specs.rb",
+        "line" => 5
+      },
+        "started_at" => "<cleaned for test>",
+       "finished_at" => "<cleaned for test>",
+           "lineage" => [
+        [0] "a successful group",
```

Comment:
💭 what's up with these indices? Should the tests spit out json. (I think it's fine as it is but the pretty printing is a little surpring

Reply:
We can make it spit out json